### ISSUE TITLE
[Feature Request] Highlight Bookmarks Inline

### ISSUE THREAD

**Issue by pyramid3d** [foobnix/LibreraReader#724]
I was using CoolReader before and I loved the feature that may bookmarks were highlighted inline when scrolling text.
This feature is very handy and I would request it for Librera.
PS: Librera is the best book reader out there. Appreciate the effort put into making such a wonderful app!

**Comment by therussellhome** [comment]
A partial solution would be to add an option to filter the bookmark ribbon for bookmarks on the current page.  

Agreed with @chemicalcain that this would be more useful when tied with #1144

**Comment by JoeyRxy** [comment]
I was using Apple Book on mac, it's so convenient to highlight a word and not lose it when change font style. I've notice google play books also has this feature. 
I think this feature is urgently needed. although you can add bookmarks for sentences, it's related to the page not sentence, and those bookmarks will be out of order if change font or style. 
Maybe there's a way to locate a paragraph instead of a page. that will help.

**Comment by nutzlichsein** [comment]
I agree with the other suggesters, this great app need this feature to be able to highlight text with color. This inline text highlight feature helps me when reading using Apple Books on my mac.

Thank you for the great app!

**Comment by y-r-v** [comment]
I fully support this request and believe inline text highlighting would significantly enhance the reading experience on Librera. In fact, it's the only feature I really miss. Similar apps handle this feature very effectively. For many people, the act of reading accompanies the act of taking notes and highlighting text.

This feature is particularly helpful for studying or revisiting important sections of books.

I understand the complexity of implementing such functionality, but it would be a game-changer for many users, myself included. Thank you for the great app!

**Comment by Appliumus** [comment]
I belive i tried ever libre ebook reader out there and have to say that i love this one. The only thing i miss is that the bookmarks i make are getting highlighted. But thanks for the development!

**Comment by frasty** [comment]
Please, add this feature. This is a must have for a book reading app. I would pay for the pro version if you plan to add this feature there.

**Comment by simurq** [comment]
As a multilingual reader, who often saves unknown words and phrases through bookmarks for later reference, I commend the previous requests under this thread. 👍🏼👍🏼👍🏼 

Currently, since tapping on a bookmark doesn't highlight it in the text (even worse when changing the reading mode, you land on a different page), one has to search the word to find and get it highlighted, which means extra effort on reader's part.

**Comment by simurq** [comment]
Any news on this one? I can see a lot of updates these days, hence the ~bump~ question. Thanks! 🙏🏼

**Comment by foobnix** [comment]
Sorry, this feature is not possible with current librera core implementation :(
